src/features/MakeOrder/BookingSuccess/ui.tsx

```diff
@@ -21,7 +22,8 @@ export const BookingSuccess: FC<TBookingSuccess> = ({ productName, daterange, me
       </StSubtitle>
       <StFlex $flexDirection='column' $alignItems='center' $gap={52}>
         <StTextBox $fontSize={0.67} $lineHeight={1} $textAlign='center'>
-          {productName} забронирован с {daterange[0]} по {daterange[1]}
+          {productName} забронирован с {toRuDateString(daterange[0])} по{' '}
+          {toRuDateString(daterange[1])}
```

Comment:
На страничке читается плохо. Может добавить переводы строки?
```suggestion
          {productName} забронирован<br />
          с {toRuDateString(daterange[0])}<br />
          по {toRuDateString(daterange[1])}
```